libs/utils/src/id.rs

```diff
@@ -20,9 +21,64 @@ pub enum IdError {
 ///
 /// Use `#[serde_as(as = "DisplayFromStr")]` to (de)serialize it as hex string instead: `ad50847381e248feaac9876cc71ae418`.
```

Comment:
Here is a comment which needs to be cleaned up, or re-written. Cannot suggest.

Reply:
Cleaned up the comments https://github.com/neondatabase/neon/pull/5335/commits/bf0078a824544ca8a6cf78ab0383a23db01839d1